src/app/results/page.tsx

```diff
@@ -1,14 +1,14 @@
 'use client'
 import { useEffect, useState } from 'react'
-import LinkButton from '../../components/linkButton'
+import LinkButton from '../../components/linkButton/linkButton'
```

Comment:
I don't think we need to double nest the linkButton here. I see what you were doing with the separate components of the modal going in a `modal` components subdirectory, but we only have a `linkButton` component so it is file to stay at the top level of the `components` directory.